patches/0013-p256-armv8-optimizations

```diff
@@ -2429,20 +2457,460 @@ Index: aws-lc/third_party/boringssl/crypto/fipsmodule/ec/p256-x86_64_test.cc
      !defined(OPENSSL_SMALL) && !defined(BORINGSSL_SHARED_LIBRARY)
  
  TEST(P256_X86_64Test, SelectW5) {
-@@ -98,6 +97,7 @@ TEST(P256_X86_64Test, SelectW7) {
-   CHECK_ABI(ecp_nistz256_select_w7, &val, table, 42);
+   // Fill a table with some garbage input.
+-  alignas(64) P256_POINT table[16];
++  __attribute__((aligned(64))) P256_POINT table[16];
```

Comment:
Why is this necessary?

Reply:
The change of attribute was needed to fix the failing tests of Linux on ARM due to an older version of gcc, older than 9.3, that doesn't have the check on alignas() in C++ code removed.
As for the alignment itself, there is this explanation in `ecp_nistz256_windowed_mul()` in `p256-x86_64.c`:
```
// A |P256_POINT| is (3 * 32) = 96 bytes, and the 64-byte alignment should
+  // add no more than 63 bytes of overhead. Thus, |table| should require
+  // ~1599 ((96 * 16) + 63) bytes of stack space.
+  alignas(64) P256_POINT table[16];
```

Reply:
Thanks.